### PR TITLE
feat: add apisix

### PR DESCRIPTION
**Pull request by NSObjects**
apisix helm chart

### CODE REVIEW

File: src/main/apisix-dashboard/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.3.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: 2.9.0
+
+maintainers:
+  - name: tao12345666333
```
Comment (zheng1) [inline src/main/apisix-dashboard/Chart.yaml:42]: Hi, @NSObjects can you submit the maintainer's email?

Comment (NSObjects) [reply src/main/apisix-dashboard/Chart.yaml:42]: ok

Comment (zheng1) [reply src/main/apisix-dashboard/Chart.yaml:42]: Now there are two items here, and these two need to be merged

**Pull request merged**